### PR TITLE
web: remove `.theme-redesign` selector from different components

### PR DESCRIPTION
**Pull request by 5h1rU**
## Changes

- Removed `.theme-redesign` usage.

Part of https://github.com/sourcegraph/sourcegraph/issues/20847.

### CODE REVIEW

File: client/web/src/tree/FileDecorator.scss [hunk 1]
```diff
@@ -4,10 +4,7 @@
         color: var(--text-muted);
 
         &--active {
-            color: var(--white);
-        }
-
-        .theme-redesign &--active {
+            color: $white;
             color: var(--text-muted);
```
Comment (tjkandala) [inline client/web/src/tree/FileDecorator.scss:8]: Why is `color` defined twice?

**Review by tjkandala — APPROVED** [review]

File: client/web/src/nav/VersionContextDropdown.scss [hunk 2]
```diff
@@ -62,6 +57,7 @@
         min-height: fit-content;
         max-height: 20rem;
         overflow-y: auto;
+        background-color: var(--dropdown-bg) !important;
```
Comment (limitedmage) [inline client/web/src/nav/VersionContextDropdown.scss:60]: Why is this added?

**Review by limitedmage — APPROVED** [review]

File: client/web/src/tree/FileDecorator.scss (hunk 1 above)
Comment (5h1rU) [reply client/web/src/tree/FileDecorator.scss:8]: nice catch!

File: client/web/src/nav/VersionContextDropdown.scss (hunk 2 above)
Comment (5h1rU) [reply client/web/src/nav/VersionContextDropdown.scss:60]: It is not needed. At some point, the background was showing me a different color, now it seems is fine.

**Review by valerybugakov — APPROVED** [review]

**Review by umpox — APPROVED** [review]
